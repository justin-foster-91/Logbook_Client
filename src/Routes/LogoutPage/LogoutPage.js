import React, { useEffect } from 'react';
import TokenService from '../../Services/token-service'

const LogoutPage = (props) => {

  const handleLogoutSuccess = () => {
    // TokenService.clearAuthToken()
    const { history } = props
    // history.push('/')
    console.log("on logout page");
    if(props.setIsLoggedIn){
      console.log("inside logout if");
      props.setIsLoggedIn(false);
    }
  }

  // handleLogoutSuccess()

  return (
    <div>
      <h1>{"Hello Logout Page"}</h1>
    </div>
  );
};

LogoutPage.defaultProps = {
  location: {},
  history: {
    push: () => {},
  },
}

export default LogoutPage;