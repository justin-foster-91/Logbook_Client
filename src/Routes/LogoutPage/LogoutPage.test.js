import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import LogoutPage from './LogoutPage';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BrowserRouter><LogoutPage /></BrowserRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
})

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<BrowserRouter><LogoutPage /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});