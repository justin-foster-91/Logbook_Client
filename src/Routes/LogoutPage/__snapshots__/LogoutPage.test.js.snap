// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <h1>
    Logout Page
  </h1>
  <p>
    How did you even get here? That's not supposed to happen.
  </p>
</div>
`;
